dist/src/main/java/io/camunda/zeebe/shared/PrometheusRegistryConfiguration.java

```diff
@@ -37,7 +35,6 @@ public CollectorRegistry collectorRegistry() {
     (new GarbageCollectorExports()).register(registry);
     (new ThreadExports()).register(registry);
     (new ClassLoadingExports()).register(registry);
-    (new VersionInfoExports()).register(registry);
```

Comment:
:question: Is the `jvm_info` metric still exported, just via some other auto configurable bean?

Reply:
Yes, it is still exported but now by default via Spring Boot's [`JvmMetricsAutoConfiguration#jvmInfoMetrics()`](https://github.com/spring-projects/spring-boot/blob/dcb41b03a032f2dd598cf005b73a0dfc966bf6cc/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/JvmMetricsAutoConfiguration.java#L77-L81).